.travis.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   - stage: lint-check
     script:
     - lein cljfmt check
-    - lein kibit
+#    - lein kibit
```

Comment:
Don't just comment it like that. Remove it.

Reply:
I think there is no need to remove this here, I could replace (when-not (empty? coll) ...) with (when (seq coll) ...) just to satisfy kibit. We must enforce a practice that we don't change the code when merging to a main branch. Let the change come from a non-merge commit. What do you think?